app/routes/dashboards/dashboard.js

```diff
@@ -1577,14 +1596,19 @@ export default Ember.Route.extend({
                         facetDash: "search",
                         facetDashParameter: "contributors",
                         dataType: 'contributors',
-                        post_body : {},
+                        post_body : {
```

Comment:
Stray whitespace before the colon. (I know this was here before the PR but would you mind fixing it?)